docs/specs/clients/notify/data-structures.md

```diff
@@ -18,13 +18,18 @@
 
 ## Notify Message
 
-```jsonc
+```typescript
 {
-  "title": string, // short message used in the title of the notification
-  "body": string, // long messages ued in the body of the notification
-  "icon": string, // image URL used to display with the notification
-  "url": string, // redirect URL for call-to-action related to notification
-  "type": string, // notification type which matches the scope of notify subscription
+  // Notification type which matches the scope of notify subscription
+  type: string,
+  // Short message used in the title of the notification
+  title: string,
+  // Long messages used in the body of the notification
+  body: string,
+  // Image URL used to display with the notification
+  icon: string | null,
```

Comment:
```suggestion
  icon?: string,
```

I believe [this](https://specs.walletconnect.com/2.0/specs/clients/notify/client-sdk-api) is the format we use

Reply:
For dev-facing things, yes, but here this is an internal data structure so the field should be nullable rather than optional.